offchain-modules/lib/src/monitor/account_monitor.rs

```diff
@@ -0,0 +1,125 @@
+        let ckb_privkeys = force_config.get_ckb_private_keys(&network)?;
+        let eth_privkeys: Vec<String> = force_config.get_ethereum_private_keys(&network)?;
+
+        let mut eth_addresses: Vec<Address> = vec![];
+        for eth_privkey in eth_privkeys.into_iter() {
+            let eth_private_key =
+                ethereum_types::H256::from_slice(hex::decode(eth_privkey)?.as_slice());
+            let eth_key = SecretKey::from_slice(&eth_private_key.0)?;
+            let from = secret_key_address(&eth_key);
+            eth_addresses.push(from);
+        }
+        let mut ckb_lockscripts: Vec<Script> = vec![];
+        for ckb_privkey in ckb_privkeys.into_iter() {
+            let privkey = get_secret_key(&ckb_privkey)?;
+            let lockscript = parse_privkey(&privkey);
+            ckb_lockscripts.push(lockscript);
+        }
+        Ok(AccountMonitor {
+            web3_client: Web3Client::new(eth_rpc_url),
+            generator: Generator::new(ckb_rpc_url, ckb_indexer_url, Default::default())
+                .map_err(|e| anyhow!("failed to crate generator: {}", e))?,
+            eth_addresses,
+            ckb_lockscripts,
+            alarm_url,
+            ckb_alarm_balance,
+            eth_alarm_balance,
+            eth_balance_conservator,
+            ckb_balance_conservator,
+        })
+    }
+
+    pub async fn start(&mut self) -> Result<()> {
+        let mut msg = " ".to_string();
+        let eth_decimal: U256 = U256::from(10u128.pow(18));
+        for eth_address in self.eth_addresses.iter() {
+            let balance = self
+                .web3_client
+                .client()
+                .eth()
+                .balance(*eth_address, None)
+                .await?
+                .div(eth_decimal);
+            let mut eth_balance_msg = format!(
+                "{} balance is : {:?} eth",
+                hex::encode(eth_address),
+                balance
+            );
+            if balance.as_u64() < self.eth_alarm_balance {
+                eth_balance_msg = format!("{} @{}", eth_balance_msg, self.eth_balance_conservator)
+            }
+            log::info!("eth balance info : {}", eth_balance_msg);
+            msg = format!("{} {} %0A", msg, eth_balance_msg);
+        }
+
+        for ckb_lockscript in self.ckb_lockscripts.iter() {
+            let live_cells = get_all_live_cells_by_lockscript(
+                &mut self.generator.indexer_client,
+                ckb_lockscript.clone(),
+            )
+            .map_err(|err| anyhow!(err))?;
+            let mut capacity = 0;
+            capacity += live_cells
+                .iter()
+                .map(|c| c.output.capacity.value())
+                .sum::<u64>()
+                .div(10u64.pow(8));
+
+            let mut ckb_balance_msg = format!(
+                "{:x} balance is : {:?} ckb",
+                ckb_lockscript.as_reader().calc_script_hash(),
```

Comment:
Better to get the ckb-address rather than ckb-lockscript-hash, if the config.default_network == mainnet, then resolve the mainnet address, otherwise, resolve the testnet address